extension/src/config/init/ensure-resources.js

```diff
@@ -14,9 +15,10 @@ function ensureCustomTypes () {
 }
 
 function ensureResources () {
+  if (config.disableEnsureResources) return Promise.resolve()
```

Comment:
Why not have this statement where ensure resources is being called? Seems confusing if you see in one place that function `ensureResources` is called but internally does not do that, wdyt?

Reply:
I prefer it here personally, as I don't think the caller of this method should worry about whether the resources actually get created or not. The end result should be the same either way, the resources exist. By disabling ensure resources, you are saying that you are managing them yourself essentially, and so they should still exist.